src/components/TransactionList/TransactionListFilters/TransactionListFiltersTs.ts

```diff
@@ -70,4 +82,22 @@ export class TransactionListFiltersTs extends Vue {
     public downloadTransactions() {
         this.$emit('downloadTransactions');
     }
+    onSelectBlackListed() {
```

Comment:
1. newline after brace
2. if we have unit tests for this component, we should add some for the new functionality